test/quantization/test_quantize_fx.py

```diff
@@ -350,9 +350,11 @@ def forward(self, x):
         original_ref_m.conv2.weight = torch.nn.Parameter(original_m.standalone.conv.weight.detach())
         original_ref_m.conv2.bias = torch.nn.Parameter(original_m.standalone.conv.bias.detach())
 
-        qconfig_dict = {'': default_qconfig, 'standalone_module_name': ['standalone']}
+        qconfig_dict = {"": default_qconfig}
+        prepare_custom_config_dict = {"standalone_module_name": ["standalone"]}
```

Comment:
Can we call this quantization_custom_config instead so that the name is clearer?